### ISSUE TITLE
Minidot doing a self alignment and --no-self not working?

### ISSUE THREAD

**Issue by cmdcolin** [thackl/minidot#7]
Hello! Thanks for the great tool. I was trying to align one fasta against another, but it seems like it is doing a self alignment despite adding the --no-self option or the -S option

I basically just have something like

    bin/minidot -S -o output.pdf A.fa B.fa

Any tips?

**Comment by cmdcolin** [comment]
I guess to be clear, the thing that I was seeing is that minimap was running on "minimap /tmp/output.fa /tmp/output.fa" and crashing. I guess from the code it looks more like --no-self is an argument to the rendering script. Is the self alignment needed to create a dotplot of just two genomes?

**Comment by thackl** [comment]
You are absolutely right, something is going wrong there. I will try to have a look at it and fix it, but it might take a while - I'm currently quite busy...
